### ISSUE TITLE
Quick connect combobox selection fix

### ISSUE THREAD

**Issue by mamaich** [mRemoteNG/mRemoteNG#1379]
mRemoteNG does not remove text selection highlighting from the quick connect combobox in some cases. This confuses as you think that the combobox has focus while it does not.

Steps to reproduce:
- enter server name in quick connect combobox and press Enter
- the server name kept is highlited in quick connect toolbar, while the focus is in another window. This is confusing.

Fix:
Edit QuickConnectComboBox.cs OnKeyDown function. After lines:
```
            if (e.KeyCode == Keys.Enter)
            {
                // Only connect if Enter was not pressed while the combo box was dropped down
                if (!_ignoreEnter)
                {
                    OnConnectRequested(new ConnectRequestedEventArgs(_comboBox.Text));
```
add line:
```
                    SelectionLength = 0;
```
so the code would look like:
```
            if (e.KeyCode == Keys.Enter)
            {
                // Only connect if Enter was not pressed while the combo box was dropped down
                if (!_ignoreEnter)
                {
                    OnConnectRequested(new ConnectRequestedEventArgs(_comboBox.Text));
                    SelectionLength = 0;
                }
...
This would remove the selection highlighting.

Please add this fix to your code.

**Comment by walchst** [comment]
I use the Quick Connect Toolbar a lot and have the same issue. I can't vouch for the code as I'm not a coder but would be great if this could be resolved!

**Comment by sparerd** [comment]
Found the cause of the issue. Turns out validation was turned on for that control for some reason, which prevented the control from naturally giving up its focus. Since we aren't performing any validation on this control anyways, I've turned that off. This fix will be in the next v1.77 alpha

**Comment by sparerd** [comment]
Related stackoverflow, for the curious: https://stackoverflow.com/questions/290629/cannot-tab-out-of-databound-winforms-dropdown-list
